### ISSUE TITLE
Improve description of --add-ignore option

### ISSUE THREAD

**Issue by cdeil** [PyCQA/pydocstyle#231]
I think the current `--add-ignore` option is a copy of `--add-select`, and it should be changed from "errors to check" to "errors to ignore", no?
```
  --add-select=<codes>  amend the list of errors to check for by specifying
                        more error codes to check.
  --add-ignore=<codes>  amend the list of errors to check for by specifying
                        more error codes to ignore.
```

An unrelated suggestion concerning:
```
$ pydocstyle --convention=numpy --ignore=D410
Cannot pass both ignore and convention. They are mutually exclusive.
```
I would have found it useful if that message had this at the end:
```
You can use `--add-ignore` instead of `--ignore`.
```

**Comment by cdeil** [comment]
Is it possible to select just one given code with the number convention?

This doesn't work:
```
$ pydocstyle --select=D412 --convention=numpy
Cannot pass both select and convention. They are mutually exclusive.
```

**Comment by Nurdok** [comment]
You can do `pydocstyle --convention=numpy --add-select=D412`.

**Comment by cdeil** [comment]
@Nurdok - What `pydocstyle --convention=numpy --add-select=D412` does is to add D412 to a list of default codes, i.e. in the report I get D412 but a lot of other things as well.

I want only `D412`, no other codes.

**Comment by Nurdok** [comment]
Oh, so just do `pydocstyle --select=D412`.

**Comment by Nurdok** [comment]
Looking back at this, it looks like the flag description is precise, but perhaps non-intuitive:

```
  --select=<codes>      choose the basic list of checked errors by specifying
                        which errors to check for (with a list of comma-
                        separated error codes or prefixes). for example:
                        --select=D101,D2
  --ignore=<codes>      choose the basic list of checked errors by specifying
                        which errors to ignore (with a list of comma-separated
                        error codes or prefixes). for example:
                        --ignore=D101,D2
  --convention=<name>   choose the basic list of checked errors by specifying
                        an existing convention. Possible conventions: pep257,
                        numpy
  --add-select=<codes>  amend the list of errors to check for by specifying
                        more error codes to check.
  --add-ignore=<codes>  amend the list of errors to check for by specifying
                        more error codes to ignore.
```
The phrasing was done to emphasize the point that the `--add-XXX` flags only _amend_ the list. I'm not sure what a better per-flag description can be, or how to make this more clear. Perhaps we just need to add a longer, general description of error code selection in the `--help` text?

**Comment by sigmavirus24** [comment]
Idea: We're all programmers here using Python so we should all be familiar with appending to lists. Perhaps those descriptions could say `append to the list of errors ...` Alternatively, `extend the list of errors ...`?

**Comment by FarmerSez** [comment]
Solved by #283 .
